cmd/faFilter/faFilter_test.go

```diff
@@ -11,20 +12,27 @@ var FaFilterTests = []struct {
 	inputFile    string
 	outputFile   string
 	expectedFile string
+	binExpected  string
 	name         string
 	notName      string
 	refPositions bool
 	start        int
 	end          int
 	minSize      int
+	binFasta     int
+	keepWhole    bool
 }{
-	{"testdata/minSizeTest.fa", "testdata/minSizeOutput.fa", "testdata/minSizeExpected.fa", "", "", false, 0, -1, 10},
+	{"testdata/minSizeTest.fa", "testdata/minSizeOutput.fa", "testdata/minSizeExpected.fa", "testdata/binExpected.fa", "", "", false, 0, -1, 10, 2, true},
+}
+var binOutputs = []string{
+	"testdata/minSizeOutput.0.fa",
+	"testdata/minSizeOutput.1.fa",
```

Comment:
Am I understanding correctly that the output after using the binFasta option should be multiple fasta files, but those multiple output fasta files are just not in cmd/faFilter/testdata because you compare them to the existing 1 expected output fasta file (binExpected.fa) and remove them?

Reply:
yes, they are compared to make sure they are being written properly and then removed since they are produced by the test run, we try to remove any files that are produced in this way so that we aren't creating a ton of extra stuff any time we test everything